### PR TITLE
[IMP] web,web_settings_dashboard: Move 'About' menu to settings dashb…

### PR DESCRIPTION
**Pull request by tivisse**
…oard

Fp request: In the user preferences menu, remove "About Us" and put the version
number in the Settings dashboard somewhere, as well as the link to switch to
developer mode. Odoo is not a desktop application anymore.

### CODE REVIEW

File: addons/web/static/src/js/framework/session.js [hunk 1]
```diff
@@ -321,7 +321,7 @@ var Session = core.Class.extend(mixins.EventDispatcherMixin, {
         var shadow = options.shadow || false;
         options.headers = _.extend({}, options.headers)
         if (odoo.debug) {
-            options.headers["X-Debug-Mode"] = true;
+            options.headers["X-Debug-Mode"] = $.deparam($.param.querystring()).debug;
```
Comment (tde-banana-odoo) [inline addons/web/static/src/js/framework/session.js]: Shouldn't it be put in a separate commit ?

Comment (tivisse) [inline addons/web/static/src/js/framework/session.js]: It sure is.
